api/envoy/extensions/filters/http/jwt_authn/v3/config.proto

```diff
@@ -60,10 +60,25 @@ message JwtProvider {
   // Specify the `principal <https://tools.ietf.org/html/rfc7519#section-4.1.1>`_ that issued
   // the JWT, usually a URL or an email address.
   //
+  // It is optional. If specified, it has to match the *iss* field in JWT.
+  //
+  // If a JWT has *iss* field and this field is specified, they have to match, otherwise the
```

Comment:
For the non-empty issuer case, have any semantics actually changed, or is this documentation change purely about clarifying existing behavior?

Reply:
Not semantics change for non-empty issuer case. These comment just clarify the existing behavior